module/armor/fan_armor.cpp

```diff
@@ -1,5 +1,24 @@
 #include "fan_armor.hpp"
 
+#include <vector>
+
 namespace fan_armor {
+FanArmor::FanArmor() {}
+
+FanArmor::~FanArmor() {}
```

Comment:
Consider explicitly using default initialization:

```suggestion
FanArmor::FanArmor() = default;

FanArmor::~FanArmor() = default;
```

Reply:
Agree